### PR TITLE
Origin/1205 convert ng model

### LINKED ISSUE #1205: Update our ngModel examples to FormControl
As part of the Angular 8 update, we have a significant number of examples using `ngModel`, which is no being phased out in favor of `FormControl`.  We want our examples to represent best practices for Angular development, so we should adjust to conform to the new form standard.

### PR DESCRIPTION
**Pull request by CameronAFaust**
Coverts all NgModels to use FormControl. 

closes #1205

### CODE REVIEW

**Comment by andrew-frueh** [comment]
Awesome @CameronAFaust @CristianIxcamey @Victor1818 - great work getting this wrapped up today.  I'll give it a look over - and let's plan on reviewing on our standup tomorrow.

**Comment by andrew-frueh** [comment]
Other than that issue with the tabs-vertical example, I think we're good to go.  You'll just need to resolve those merge collisions on datepicker-sugar and pagination-simple. Let's go over that on our call today, but in a nutshell - from your branch:
`git fetch upstream`
`git merge upstream/dev`
Then VSCode will bring up the merge collisions, and you'll just need to decide how to resolve them.

**Comment by andrew-frueh** [comment]
@corykon - if you're still checking email today and you can just give them changes from the Neumont team a quick look over, that would be awesome.  I've all the examples and everything is working as expected, but would feel better if you gave the code changes a 👍

File: projects/cashmere-examples/src/lib/pagination-simple/pagination-simple-example.component.html [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+    <hc-label>Display width:</hc-label>
+    <hc-select [formControl]="widthControl">
+        <option value="lg">Large</option>
+        <option value="md">Medium</option>
+        <option value="sm">Small</option>
+    </hc-select>
+</hc-form-field>
 
-<hc-pagination [(pageNumber)]="pageNumber" [(pageSize)]="pageSize" [length]="totalItemsControl.value"></hc-pagination>
+<hc-pagination [(pageNumber)]="pageNumber" [(pageSize)]="pageSize" [length]="totalItemsControl.value" [displayWidth]="widthControl.value">></hc-pagination>
```
Comment (andrew-frueh) [inline projects/cashmere-examples/src/lib/pagination-simple/pagination-simple-example.component.html:13]: I think you might have ended up with one extra `>` at the end of your first `hc-pagination` tag

Comment (CameronAFaust) [reply projects/cashmere-examples/src/lib/pagination-simple/pagination-simple-example.component.html]: This should be fixed now.

**Comment by andrew-frueh** [comment]
Shoot - @CameronAFaust looks like the linting is failing because the commit message wasn't properly formatted.  Would it be possible to roll back that comment, recommit with proper formatting, and then force push up to the branch?

**Comment by andrew-frueh** [comment]
Thanks for the review @corykon and thanks for making those final changes @CameronAFaust.  Let's get this one merged and push a new build!

**Pull request merged**